src/Propel/Runtime/Collection/OnDemandCollection.php

```diff
@@ -30,7 +30,7 @@ class OnDemandCollection extends Collection
     private $lastIterator;
 
     /**
-     * @param AbstractFormatter    $formatter
+     * @param \Propel\Runtime\Formatter\ObjectFormatter    $formatter
```

Comment:
Why not AbstractFormatter?

Reply:
it doesnt have the method we need.
and parent is already the same.
refactoring the constructor would be BC break for now.